app/src/main/java/org/mozilla/fenix/settings/creditcards/CreditCardEditorFragment.kt

```diff
@@ -50,6 +47,13 @@ class CreditCardEditorFragment : Fragment(R.layout.fragment_credit_card_editor)
         )
     }
 
+    override fun onResume() {
+        requireActivity().window.setFlags(
```

Comment:
nit: we seem to use `addFlags`/`clearFlags` in other places, might be best to follow the same pattern there too.